components/dayspan-custom-calendar.vue

```diff
@@ -325,6 +327,14 @@ export default {
     canAddTime()
     {
       return !this.readOnly
+    },
+
+    ...mapGetters({
+      getHasLecturesOnWeekend: 'schedule/getHasLecturesOnWeekend',
```

Comment:
store getters = "computed properties für den Store" (https://vuex.vuejs.org/guide/getters.html), anders als in Java.
Direkt auf den state zuzugreifen ist auch okay (https://codeburst.io/vuex-getters-are-great-but-dont-overuse-them-9c946689b414).

Reply:
Geändert